sdcm/sla/libs/sla_utils.py

```diff
@@ -230,21 +230,21 @@ def validate_scheduler_runtime(self, start_time, end_time, read_users, prometheu
             #                             f'Run time per role: {roles_full_info}')
```

Comment:
`Run time` here too.

Reply:
this code was commented in 2022. decided not to touch it for now